ash/src/vk/features.rs

```diff
@@ -54,6 +45,14 @@ impl StaticFn {
         (self.get_instance_proc_addr)(instance, p_name)
     }
 }
+#[derive(Clone, Debug)]
+pub struct MissingEntryPoint(pub std::ffi::CString);
+impl std::fmt::Display for MissingEntryPoint {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
+        write!(f, "missing Vulkan entry point")
```

Comment:
Should perhaps include `self.0` in the format so that the caller knows what failed to load? If reserving `MissingEntryPoint` purely for the `vkGetInstanceProcAddr` "entrypoint" (which should be the only dynamic symbol in a Vulkan library) it is perhaps better to remove that member entirely.

Reply:
I took some inspiration from Vulkano on this, and figured there may be future cases where we want to include other function names. Then that can be changed without breaking anyone's code.

Reply:
Sounds good, but don't forget to print it in `write!` in that case :)